www/_posts/2018-08-01-future-cordova-ios-webview.md

```diff
@@ -0,0 +1,70 @@
+
+## Cordova iOS Future Release, WKWebView only
+
+When UIWebView support has been removed in a future iOS SDK, we will aim to release a future Cordova iOS version, which will remove UIWebView support, and WKWebView will then be the default webview engine.
+
+| Cordova iOS 4 >>> | Cordova iOS 5 >>> | Cordova iOS Future Release |
+|-------------------|-------------------|---------------|
+|  | UIWebView deprecated | UIWebView removed |
+|  | WKWebView Engine Plugin Integrated Into Platform | WKWebView Engine Plugin only |
+|  | Bridge WebView Plugin Added | Bridge WebView Plugin Removed |
+
+
+## Limitations of WKWebView
+
+There are [many limitations](https://mjtsai.com/blog/2018/06/20/webview-and-uiwebview-deprecated-in-favor-of-wkwebview/) of WKWebview, especially if you were using UIWebView previously. The limitations are:
+
+1. Cookies don't persist. This is a WebKit bug, but someone has
+created a plugin for a workaround. See
+[CB-12074](https://issues.apache.org/jira/browse/CB-12074)
+2. Can't delete cookies. This is/was a WebKit bug (2015), we need to test
+for the iOS 11/12. See [CB-11297](https://issues.apache.org/jira/browse/CB-11297)
+3. Can't execute JavaScript code in the background. There are several
+issues related to this. See
+[CB-12815](https://issues.apache.org/jira/browse/CB-12815)
+4. XmlHttpRequests don't work, because of Cross-Origin Resource
+Sharing issue (CORS). There is a workaround plugin created by Oracle
+(UPL licensed, which is Apache-2.0 compatible). See
+[CB-10143](https://issues.apache.org/jira/browse/CB-10143)
+5. Migration of localStorage from UIWebView. There is a migration
+plugin available. See [CB-11974](https://issues.apache.org/jira/browse/CB-11974)
+
+There are several bugs that need to be resolved as well. The full list
+here: [https://s.apache.org/QfsF](https://s.apache.org/QfsF)
+
+As you can see, WKWebView is not a direct drop-in replacement for UIWebView, you will need several plugins to patch functionality that is missing. There is also the [local-webserver experimental plugin option](https://github.com/apache/cordova-plugins/tree/wkwebview-engine-localhost), which will not be graduated to a full plugin -- we will concentrate our efforts on supporting the main WKWebView engine plugin. 
+
+Hopefully with more testing, and filing of bug reports with Apple for missing features, the WKWebView can be a full replacement for Cordova users.
+
+
```

Comment:
Add another section with something like:

## Other breaking changes coming

- iOS pre-10 is deprecated and will be dropped in the near future
- iOS pre-11 is deprecated and will be dropped in a future release

Reply:
I think this should go in a future blog post. We've never fully qualified our support policy, except in dev@ discussions.